source/features/follow-file-renames.tsx

```diff
@@ -57,6 +58,10 @@ async function linkify(button: HTMLButtonElement, filePath: string): Promise<voi
 		// Clear the search from the url, so it does not get passed to the rename link
 		search: '',
 	});
+	if (isOlder) {
+		linkifiedURL.branch = fileChanges.commit.parentSha; // #5576
```

Comment:
How come this became necessary? The feature worked before

Reply:
I was wondering about that too, but I very rarely use this feature so I don't know. Maybe GitHub used to redirect to the last commit where the file existed, but stopped doing it for whatever reason?

Reply:
hmm that's it, if I remember correctly you could use the commits page on any file and it would just show the commits whether the file existed or not, which would explain why this worked.